python/private/local_runtime_repo.bzl

```diff
@@ -46,7 +46,7 @@ def _local_runtime_repo_impl(rctx):
     on_failure = rctx.attr.on_failure
 
     platforms_os_name = repo_utils.get_platforms_os_name(rctx)
-    if not platforms_os_name:
+    if platforms_os_name not in ["linux", "osx", "windows"]:
```

Comment:
Just remove this whole block. I had it return None and bail out because I knew there were some unhandled cases, but didn't know what they were or how to map them.